spec/lib/submission_builder/ty2024/states/nj/nj_return_xml_spec.rb

```diff
@@ -164,5 +164,49 @@
         end
       end
     end
+
+    describe "additional dependents PDF" do
+      context "when there are more than 4 dependents" do
+        let(:intake) { create(:state_file_nj_intake, :df_data_minimal) }
+        let(:nj_return) { described_class.new(submission) }
+
+        before do
+          create :state_file_dependent, intake: intake
```

Comment:
[dust] This could be simplified to `5.times { create :state_file_dependent, intake: intake } ` here and below. Having the actual number specified might improve readability in this case